SCons/Util.py

```diff
@@ -31,8 +31,15 @@
 import sys
 from collections import UserDict, UserList, UserString, OrderedDict
 from collections.abc import MappingView
+from contextlib import suppress
 from types import MethodType, FunctionType
 
+# Note: Util module cannot import other bits of SCons globally without getting
+# into import loops. Both the below modules import SCons.Util early on.
```

Comment:
Which "both below modules" modules?

Reply:
SCons.Errors and SCons.Warnings - the two tcommented out imports

Reply:
Gotcha